### PR TITLE
ci: add integration test with 61s sleep

### PR DESCRIPTION
**Pull request by jamilbk**
Refs #4484

### CODE REVIEW

File: scripts/tests/direct-curl-heartbeat-timeout.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source "./scripts/tests/lib.sh"
+
+sleep 70 # Ensure a couple heartbeats have allegedly elapsed
```
Comment (jamilbk) [inline scripts/tests/direct-curl-heartbeat-timeout.sh]:
```suggestion
sleep 61 # Ensure a couple heartbeats have allegedly elapsed
```

**Review by thomaseizinger — APPROVED** [review]
I am not sure this actually catches anything. At the same time, we don't currently have any CI test that simulates something like a long-running job so this is an improvement.

**Review by AndrewDryga — APPROVED** [review]

**Comment by jamilbk** [comment]
> I am not sure this actually catches anything.

Yeah, since the portal doesn't close websockets now until 307 seconds, we would need to run this test for that long.

**Comment by jamilbk** [comment]
Also if we remove the first curl before the sleep, it would check that the Relay connection isn't being cut by the portal.

**Comment by jamilbk** [comment]
Closing this because I think soak tests are better left for #4506

**Pull request closed without merging**